File: Emlak/src/app/admin/model/ITipler.ts
export interface ITipler {
    ID: number,
    //[Required(ErrorMessage = "Tip Ýsmi alaný boþ olamaz ve en fazla 50 karakter olmalýdýr.")]
    //[StringLength(50)]
    TypeName: string,
    //[Required(ErrorMessage = "Url alaný boþ olamaz ve en fazla 100 karakter olmalýdýr.")]
    //[StringLength(100)]
    Url: string,
    //[Required(ErrorMessage = "Tablo Ýsmi alaný boþ olamaz ve en fazla 50 karakter olmalýdýr.")]
    //[StringLength(50)]
    TableName: string,
    Linkable: boolean,
    Show: boolean,

    Mesaj: string
}

File: AdminPanel/Repository/Tipler/ITipler.cs
using System.Collections.Generic;

namespace Repository.TiplerModel
{
    public interface ITipler
    {
        int ID { get; set; }
        string TypeName { get; set; }
        string Url { get; set; }
        string TableName { get; set; }
        bool Linkable { get; set; }
        bool Show { get; set; }

        string Mesaj { get; set; }

        List<Tipler> List();
        ITipler Select(int id);
        bool Insert(ITipler tip);
        bool Update(ITipler tip);
        bool Delete(int id);
    }
}
